OWNERS

```diff
@@ -1,28 +1,31 @@
 approvers:
-  - dtrifiro
+  - danielezonca
+  - terrytangyuan
```

Comment:
I think we can remove Yuan

Reply:
I keep them in approvers list only because Daniele is our Architect and Yuan is an leading Model Server team. It would allow them to run PR checks apart from our internal team in case our team is not reachable.